app.py

```diff
@@ -418,8 +429,8 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
             collateral_usd_amounts,
             values='amount_usd',
             names='token',
-            title='Collateral (USD)',
-            color_discrete_sequence=plotly.express.colors.sequential.Oranges_r,
+            title='Collateral (USD)',  
+            color_discrete_sequence=[color_map.get('Collateral')],
```

Comment:
You changed `COLORS_MAP` but didn't change `color_map`
```suggestion
            color_discrete_sequence=[COLORS_MAP.get('Collateral')],
```